### PR TITLE
Create ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58128](https://cumulocity.atlassian.net/browse/MTM-58128)
Original PR: [5721](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5721)

### CODE REVIEW

File: content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/y2024] (#5721)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix inability to switch tabs with multiple widgets using dashboard date context
```

File: content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58128
+version: 1018.503.54
+---
+Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/y2024] (#5721)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:17]: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where users were unable to switch between dashboard tabs if the dashboard contained two or more widgets using the dashboard date context filter. Users can now smoothly transition between tabs regardless of the number of dashboard date context widgets present.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix inability to switch tabs with multiple widgets using dashboard date context
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-54-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md:3]:
```suggestion
title: Switching tabs with multiple widgets using the dashboard date context works smoothly
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**